test/generators/suspenders/jobs_generator_test.rb

```diff
@@ -0,0 +1,176 @@
+require "test_helper"
+require "generators/suspenders/jobs_generator"
+
+module Suspenders
+  module Generators
+    class JobsGeneratorTest < Rails::Generators::TestCase
+      include Suspenders::TestHelpers
+
+      tests Suspenders::Generators::JobsGenerator
+      destination Rails.root
+      setup :prepare_destination
+      teardown :restore_destination
+
+      test "raises if API only application" do
+        within_api_only_app do
+          assert_raises Suspenders::Generators::APIAppUnsupported::Error do
+            run_generator
+          end
+
+          assert_file app_root("Gemfile") do |file|
+            assert_no_match "sidekiq", file
+          end
+        end
+      end
+
+      test "does not raise if API configuration is commented out" do
+        within_api_only_app commented_out: true do
+          run_generator
+
+          assert_file app_root("Gemfile") do |file|
+            assert_match "sidekiq", file
+          end
+        end
+      end
+
+      test "adds gems to Gemfile" do
+        backup_file "config/application.rb" if File.exist? app_root("config/application.rb")
+
+        expected_output = <<~RUBY
+          gem "sidekiq"
+        RUBY
+
+        run_generator
+
+        assert_file app_root("Gemfile") do |file|
+          assert_match(expected_output, file)
+        end
+      end
+
+      test "installs gems with Bundler" do
+        Bundler.stubs(:with_unbundled_env).yields
+        generator.expects(:run).with("bundle install").once
+
+        capture(:stdout) do
+          generator.add_sidekiq_gem
+        end
+      end
```

Comment:
[Minor]

After exploring the [Rails codebase](https://github.com/rails/rails/blob/37833debc669bd3435a0d70b1d294bda2eb14070/railties/test/generators/app_generator_test.rb#L647-L651) for inspiration on how they test bundle commands, I've [started to follow the same pattern](https://github.com/thoughtbot/suspenders/blob/73cdb80fb1f1bd3d11b5b87f5e00e0015206a541/test/generators/suspenders/styles_generator_test.rb#L36-L40). My hope is that we might be able to remove the `mocha` dependency.

```suggestion
      test "installs gems with Bundler" do
        output = run_generator
        
        assert_match /bundle install/, output
      end
```